integrations/tests/test_shuffle.py

```diff
@@ -0,0 +1,95 @@
+                                                        'firedtimes': 1},
+                                               'id': 'alert_id',
+                                               'full_log': 'full log.', 'decoder': {'name': 'decoder-name'},
+                                               'location': 'wazuh-X'},
+                                              '{"severity": 1, "pretext": "WAZUH Alert", "title": "alert description", "text": "full log.", "rule_id": "rule-id", "timestamp": "year-month-dayThours:minuts:seconds+0000", "id": "alert_id", "all_fields": {"timestamp": "year-month-dayThours:minuts:seconds+0000", "rule": {"level": 0, "description": "alert description", "id": "rule-id", "firedtimes": 1}, "id": "alert_id", "full_log": "full log.", "decoder": {"name": "decoder-name"}, "location": "wazuh-X"}}'),
+                                             ({'timestamp': 'year-month-dayThours:minuts:seconds+0000',
+                                               'rule': {'level': 0, 'description': 'alert description',
+                                                        'id': random.choice(filter_values),
+                                                        'firedtimes': 1},
+                                               'id': 'alert_id',
+                                               'full_log': 'full log.', 'decoder': {'name': 'decoder-name'},
+                                               'location': 'wazuh-X'},
+                                              '')])
+def test_generate_msg(json_alert, msg):
+    """
+    Test that the expected message is generated when json_alert received.
+
+    Parameters
+    ----------
+    json_alert : json
+        json data that simulates the values that could have
+        been obtained from alert_file_location.
+
+    msg : str
+        message that should be retourned by the generate_msg function
+    """
+    assert shuffle.generate_msg(json_alert) == msg
+
+
+@pytest.mark.parametrize('json_alert', [({'timestamp': 'year-month-dayThours:minuts:seconds+0000',
+                                                         'rule': {'level': 0, 'description': 'alert description',
+                                                                  'id': 'rule-id',
+                                                                  'firedtimes': 1},
+                                                         'id': 'alert_id',
+                                                         'full_log': 'full log.', 'decoder': {'name': 'decoder-name'},
+                                                         'location': 'wazuh-X'})])
+def test_filtered_msg(json_alert):
+    """
+    Test that the alerts with certain rule ids are filtered.
+
+    Parameters
+    ----------
+    json_alert : json
+        json data that simulates the values that could have
+        been obtained from alert_file_location.
+
+    """
+    json_alert['rule']['id'] = random.choice(filter_values)
```

Comment:
we should not choose a random value. This could cause the test to work on some runs and fail on others. Consider always using the same id or traversing the entire list.